pkg/osquery/table/touchid_system_darwin.go

```diff
@@ -75,8 +73,11 @@ func (t *touchIDSystemConfigTable) generate(ctx context.Context, queryContext ta
 	configOutStr := string(stdout.Bytes())
 	configSplit := strings.Split(configOutStr, ":")
 	if len(configSplit) >= 3 {
+		touchIDCompatible = "1"
 		touchIDEnabled = configSplit[2][1:2]
 		touchIDUnlock = configSplit[3][1:2]
+	} else {
+		touchIDCompatible = "0"
```

Comment:
```suggestion
```
I don't think you need this, 0 is the default value for an int. (the original code didn't need it either)